<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DangNhap</class>
 <widget class="QMainWindow" name="DangNhap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Nhập</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon/Bot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tài Khoản</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txttk">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtmk">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mật Khẩu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ĐĂNG NHẬP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDangNhap">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Đăng Nhập</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDangKy">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Đăng ký</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tài khoản mới</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txttk_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtmk_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mật Khẩu mới</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nhập lại mật khẩu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtmk_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ĐĂNG KÝ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDangKy_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Đăng ký</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDangNhap_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Quay lại</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
